File: Timer.java
public class Timer {
    public static void runTimer(int seconds) {
        for (int second = seconds; second >= 0; second--) {
            System.out.printf("Temps restant : %d secondes%n", second);
            try {
                Thread.sleep(1000); // Attente de 1 seconde (1000 ms)
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
        }
        System.out.println();
    }
}

File: timer.py
import time


class Timer:
    @staticmethod
    def run_timer(seconds):
        if seconds > 0:
            for second in range(seconds, -1, -1):
                print(
                    f"Temps restant : {second}"
                    " secondes", end='\r')
                time.sleep(1)
            print("\n")